### DISCUSSION TITLE
Dogfooding

### DISCUSSION THREAD

**Discussion by elmstedt** [kuafuai/DevOpsGPT#96] [Ideas]
This is a very interesting and exciting project.

What would really sell it though is if you could demonstrate DevOpsGPT generating DevOpsGPT, similarly to how many 3D-printer manufacturers like to claim it is possible to 3D-print a copy of their 3D-printer using their 3D-printer.

When DevOpsGPT can write code functionally equivalent to DevOpsGPT, I think that's when your project starts getting national media attention.

Good luck!

**Comment by booboosui** [comment]
You're absolutely correct, and we've been giving this a lot of thought.

A crucial element in bringing this capability to life revolves around comprehending the existing codebase. To tackle this, we're leveraging advanced trained models. We're excited to share that we'll be focusing on rigorously testing this capability, and you can expect to see it in action in our upcoming commercial release very soon.
